xml/install_caasp_heat_templates.xml

```diff
@@ -444,244 +444,275 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
+     presented by &o_ident; at the specified
+     <literal>keystone_auth_url</literal> in the <literal>System-wide
+     certificate</literal> section of Velum. The certificate must be uploaded
+     to avoid bootstrapping failure with a <literal>x509: certificate signed
+     by unknown authority</literal> error message.
+    </para>
+    <para>
+     Get SUSE Root CA certificate with the following steps:
+    </para>
+    <substeps>
+     <step>
+      <para>
+       From the RPM provided in the repos:
+      </para>
+      <substeps>
+       <step>
+        <para>
+         Add a repository for your distribution at
+         <literal>http://download.suse.de/ibs/SUSE:/CA/</literal>. For
+         example, with <literal>openSUSE Leap</literal>, use
+         <literal>http://download.suse.de/ibs/SUSE:/CA/openSUSE_Leap_15.0/SUSE:CA.repo</literal>
+        </para>
+       </step>
+       <step>
+        <para>
+         Install certificates and find the <filename>.pem</filename> file.
+        </para>
+        <screen>&prompt.ardana;sudo zypper in ca-certificates-suse
+&prompt.ardana;rpm -ql ca-certificates-suse</screen>
+        <para>
+         The path to the cert is <filename>/usr/share/pki/trust/anchors/SUSE_Trust_Root.crt.pem</filename>
+        </para>
+       </step>
+      </substeps>
+     </step>
+     <step>
+      <para>
+       In <literal>system wide certificate</literal> during setup in Velum,
+       upload this certificate so you can bootstrap with CPI.
+      </para>
+     </step>
+    </substeps>
+   </step>
+   <step>
+    <para>
+     After the nodes come up, continue with the installation instructions in
+     <xref linkend="sec.heat.templates.install"/> following the admin node
+     cluster bootstrapping steps.
+    </para>
+   </step>
+  </procedure>
+ </section>
+ <section xml:id="sec.heat.templates.register">
+  <title>Register &caasp; Cluster for Software Updates</title>
+  <para>
+   Software updates are published for all registered users of &caasp;, and
+   should always be enabled upon deploying a new cluster.
+  </para>
+  <para>
+   These steps may be performed on cluster nodes one at a time, or in parallel,
+   making SSH connections as the <literal>root</literal> user with the password
+   that was established in your <filename>/usr/share/caasp-openstack-heat-templates/caasp-environment.yaml</filename> file.
+  </para>
+  <note>
+   <para>
+    If using a private SMT server for registration, use its hostname or IP
+    address when running the commands below. Otherwise, use
+    <literal>scc.suse.com</literal> to connect to &suse;'s public registration
+    server.
+   </para>
+  </note>
+  <procedure>
+   <step>
+    <para>
+     If this node was previously registered, deactivate its current registration:
+    </para>
+    <screen>&prompt.sudo;SUSEConnect -d
+&prompt.sudo;SUSEConnect --cleanup</screen>
+   </step>
+   <step>
+    <para>
+     If you are registering with a private SMT server, install its SSL
```

Comment:
```suggestion
 If you are registering with a private SMT server, install the SSL
```

Reply:
It is definitely most correct as-is.

Reply:
Whilst "its" is a possessive form of it and this is correct, it just doesn't read seamlessly. "the" provides the same function. Up to you though. Using "its" implies that the server is capable of owning an SSL certificate. "the" indicates that the server has an SSL certificate. I've corrected another instance above too :)

Reply:
@csymons-suse - any thoughts on this one? I corrected a similar instance above on line 601.